src/main/java/com/univocity/parsers/containers/Row.java

```diff
@@ -0,0 +1,66 @@
+package com.univocity.parsers.containers;
+
+import java.util.Arrays;
+
+/**
+ * @author naveen.kasthuri
+ * A convenient container of information represented in a csv row. Instead of handling lists of
+ * arrays of strings, a csv can be fully represented by a list of rows.
+ *
+ * The row object acts as a map from header to a cell. For example: row.get("Make") will return the
+ * value of this row's cell under column "Make". However, each row object does not duplicate the
+ * keys (headers) like a typical Map would: only one copy of the keys is stored in the {@link Csv}
+ * object. As such, this is a memory efficient, computationally efficient, convenient container of
+ * row objects.
+ *
+ * @see com.univocity.parsers.examples's RowObjectExample for an implementation example
+ */
+public class Row {
+  private final String[] values;
+  private final int lineNumber;
+  private final Csv csv;
```

Comment:
The csv attribute name does not make much sense as this object is storing record metadata (i.e. header names)